core/org.eclipse.cdt.core/src/org/eclipse/cdt/core/build/ICBuildConfigurationManager.java

```diff
@@ -49,54 +48,45 @@ boolean hasConfiguration(ICBuildConfigurationProvider provider, IProject project
 			throws CoreException;
 
 	/**
-	 * Create a new build configuration to be owned by a provider.
+	 * Create a new Platform Build configuration to be owned by a Core Build config provider.
 	 *
-	 * @param provider
-	 * @param project
-	 * @param configName
+	 * @param provider The project's Core Build config provider.  Must not be null.
+	 * @param project Project in which the Platform Build configuration is created. Must not be null.
+	 * @param configName The Core Build config name to be used as part of the the Platform Build configuration
+	 * name. Must not be null.
 	 * @param monitor
-	 * @return new build configuration
-	 * @throws CoreException
+	 * @return new Platform Build configuration. Not null.
+	 * @throws CoreException Reasons include the reasons given in
+	 * {@link IProject#setDescription(org.eclipse.core.resources.IProjectDescription, IProgressMonitor)}
+	
```

Comment:
Trailing white space here causes code cleanliness fail, but further than that lines in a javadoc should start with `*`. In this case as it is the last line I am suggesting deleting the whole line.
```suggestion
```

PS You can see this in the build output [here](https://github.com/eclipse-cdt/cdt/actions/runs/13264673819/job/37029075519#step:4:134) <-- is a link to the specific line, but if you click on Details link you should come to the same approximate place:

![image](https://github.com/user-attachments/assets/7a80fb0a-dbc1-469c-8443-2d09eb9a6a04)



This is a screenshot so you can see the key parts:

![image](https://github.com/user-attachments/assets/38a25896-df60-4043-a5e0-5c3e17b5556c)

Reply:
Ah got it. Thanks for the comprehensive help on that.
I'm missing formatting in comments 😢